roles/base/templates/docker_facts.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
+exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
```

Comment:
can't we just do this for the tasks in teh ansible above, so we can get rid of the docker dependency which seems to add a lot of extra stuff?